yodeploy/cmds/install_certificates.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+import re
+import sys
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+
+from yodeploy import config
+
+_SSL_CERT_PATH = '/etc/ssl/certs'
+_SSL_PRIVATE_KEY_PATH = '/etc/ssl/private'
```

Comment:
Presumably this is going to be run as a user, putting the files in `/etc/` doesn't seem right. Somewhere in `~/.cache/` or `~/.local/share` seems more appropriate. (Like `yodeploy.config.deploy_config_paths()` does)